### PR TITLE
[solrcloud] update to solr9

### LINKED ISSUE #3780: Set up new solr cluster
set up three new VMs running solr cloud 9.n. server

### PR DESCRIPTION
**Pull request by kayiwa**
- add solr9cloud inventory
- solr 9 saves modules in a different location
- uses ruby_s role

closes #3780

### CODE REVIEW

**Comment by kayiwa** [comment]
successfully run on lib-solr-staging{1,2,3}

**Comment by kayiwa** [comment]
run successfully on lib-solr-prod{1,2,3}

File: group_vars/solr9cloud/production.yml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+bundler_version: "2.3.18"
+solr_heap_setting: '20g'
+solr_cloud_download_version: 9.2.0
+solr_log4j_path: '/solr/log4j2.xml'
+lib_zk1_host_name: lib-zk-staging1
+lib_zk2_host_name: lib-zk-staging2
+lib_zk3_host_name: lib-zk-staging3
+lib_zk1_host: '{{ vault_lib_zk_staging1_host }}'
+lib_zk2_host: '{{ vault_lib_zk_staging2_host }}'
+lib_zk3_host: '{{ vault_lib_zk_staging3_host }}'
```
Comment (rlskoeser) [inline group_vars/solr9cloud/production.yml:15]: the filename says production but these are all staging

**Review by rlskoeser — APPROVED** [review]
Looks good as far as I can tell, except the staging hosts in the production vars

**Review by rlskoeser — APPROVED** [review]

**Pull request merged**